### PR TITLE
Update the `load` command description as it supports full backups

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 A database backup artifact (full or differential) can be restored within the same or to a later Neo4j version using the `restore` command of `neo4j-admin`.
 
+Starting with Neo4j 5.20, you can load a full database backup artifact using xref:backup-restore/restore-dump.adoc#restore-dump-command[`neo4j-admin database load` command] as well.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-backup.adoc:11]:
```suggestion
Starting with Neo4j 5.20, you can load a full database backup artifact using the xref:backup-restore/restore-dump.adoc#restore-dump-command[`neo4j-admin database load`] command as well.
```

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 2]
```diff
@@ -8,6 +8,9 @@
...
+This is available in the Community Edition.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-backup.adoc:12]:
```suggestion
This functionality is available in the Community Edition.
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 3]
```diff
@@ -4,17 +4,20 @@
 
 A database dump can be loaded to a Neo4j instance using the `load` command of `neo4j-admin`.
 
+Starting with Neo4j 5.20, `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:7]:
```suggestion
Starting with Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 4]
```diff
@@ -124,6 +128,13 @@ The following is an example of how to load the dump from an AWS S3 URI, as of Ne
 bin/neo4j-admin database load --from-path=s3://full/path/to/aws-s3-dumps/ <database> --overwrite-destination=true
 ----
 
+The following example shows how to specify a specific file for the `load` command, as of Neo4j 5.20.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:131]:
```suggestion
The following example shows how to designate a specific archive for the `load` command, as of Neo4j 5.20.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 5]
```diff
@@ -124,6 +128,13 @@ The following is an example of how to load the dump from an AWS S3 URI, as of Ne
 bin/neo4j-admin database load --from-path=s3://full/path/to/aws-s3-dumps/ <database> --overwrite-destination=true
 ----
 
+The following example shows how to designate a specific archive for the `load` command, as of Neo4j 5.20.
```
Comment (zimmre) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:131]: remove `as of Neo4j 5.20` this functionality is there since 4.something

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

**Review by zimmre — APPROVED** [review]

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 6]
```diff
@@ -4,17 +4,20 @@
 
 A database dump can be loaded to a Neo4j instance using the `load` command of `neo4j-admin`.
 
+Starting from Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:7]:
```suggestion
Starting from Neo4j 5.20, the `neo4j-admin database load` command supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].

You can use this functionality for restoring full backup files created in the Aura instance (link:https://neo4j.com/docs/aura/auradb/managing-databases/backup-restore-export/).
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 7]
```diff
@@ -4,17 +4,20 @@
...
+
 [NOTE]
 ====
 Change Data Capture does **not** capture any data changes resulting from the use of `neo4j-admin database load`.
-See link:{neo4j-docs-base-uri}/cdc/current/getting-started/key-considerations/#non-tx-log-changes[Change Data Capture -> Key considerations] for more information.
+See link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#non-tx-log-changes[Change Data Capture -> Points to consider] for more information.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:12]:
```suggestion
See link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#non-tx-log-changes[Change Data Capture -> Key considerations] for more information.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 8]
```diff
@@ -8,6 +8,9 @@
 
 A database backup artifact (full or differential) can be restored within the same or to a later Neo4j version using the `restore` command of `neo4j-admin`.
 
+Starting with Neo4j 5.20, you can load a **full** database backup artifact using the xref:backup-restore/restore-dump.adoc#restore-dump-command[`neo4j-admin database load`] command.
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-backup.adoc:11]: I would leave this command out of this page, but we can discuss it.

File: modules/ROOT/pages/backup-restore/restore-dump.adoc (hunk 6 above)
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:7]:
```suggestion
From Neo4j 5.20, the `neo4j-admin database load` command also supports loading a database from xref:backup-restore/online-backup.adoc#backup-artifact[full database backup artifacts].
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 9]
```diff
@@ -4,17 +4,20 @@
...
 ====
 
 
 [[restore-dump-command]]
 == Command
 
-The `neo4j-admin database load` command loads a database from an archive created with the xref:backup-restore/offline-backup.adoc#offline-backup-command[`neo4j-admin database dump`] command.
+The `neo4j-admin database load` command loads a database from an archive created with the xref:backup-restore/offline-backup.adoc#offline-backup-command[`neo4j-admin database dump`] command or from xref:backup-restore/online-backup.adoc#online-backup-command[full Neo4j Enterprise backup] starting with Neo4j 5.20.
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:19]:
```suggestion
The `neo4j-admin database load` command loads a database from an archive created with the xref:backup-restore/offline-backup.adoc#offline-backup-command[`neo4j-admin database dump`] command or from xref:backup-restore/online-backup.adoc#online-backup-command[full Neo4j Enterprise backup] (available from Neo4j 5.20).
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 10]
```diff
@@ -37,11 +40,12 @@ neo4j-admin database load [-h] [--expand-commands] [--info] [--verbose] [--overw
 === Description
 
 Load a database from an archive.
-_<archive-path>_ must be a directory containing an archive(s) created with the dump command.
-If neither `--from-path` or `--from-stdin` is supplied `server.directories.dumps.root` setting will be searched for the archive.
-Existing databases can be replaced by specifying `--overwrite-destination`.
+_<archive-path>_ must be a directory containing an archive(s).
+Archive can be a database dump created with the dump command, or can be a full backup artifact created by the backup command from Neo4j Enterprise.
+If neither --from-path or --from-stdin is supplied `server.directories.dumps.root` setting will be searched for the archive.
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:45]:
```suggestion
If neither `--from-path` or `--from-stdin` is supplied `server.directories.dumps.root` setting will be searched for the archive.
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 11]
```diff
@@ -37,11 +40,12 @@ neo4j-admin database load [-h] [--expand-commands] [--info] [--verbose] [--overw
...
+Existing databases can be replaced by specifying --overwrite-destination.
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:46]:
```suggestion
Existing databases can be replaced by specifying `--overwrite-destination`.
```

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 12]
```diff
@@ -37,11 +40,12 @@ neo4j-admin database load [-h] [--expand-commands] [--info] [--verbose] [--overw
...
 It is not possible to replace a database that is mounted in a running Neo4j server.
-If `--info` is specified, then the database is not loaded, but information (i.e. file count, byte count, and format of load file) about the archive is printed instead.
+If --info is specified, then the database is not loaded, but information (i.e. file count, byte count, and format of load file) about the archive is printed instead.
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:48]:
```suggestion
If `--info` is specified, then the database is not loaded, but information (i.e. file count, byte count, and format of load file) about the archive is printed instead.
```

**Review by renetapopova — APPROVED** [review]
Looks good. I added some minor suggestions.

File: modules/ROOT/pages/backup-restore/restore-dump.adoc [hunk 13]
```diff
@@ -140,6 +145,20 @@ You can load a dump from a local directory using the following command:
 bin/neo4j-admin database load --from-path=/full-path/data/dumps <database> --overwrite-destination=true
 ----
 
+Starting from Neo4j 5.20, you can use the same command to load the database from its full backup artifact:
+
+[source,shell, role="nocopy"]
+----
+bin/neo4j-admin database load --from-path=/full-path/to/backups <database> --overwrite-destination=true
```
Comment (renetapopova) [inline modules/ROOT/pages/backup-restore/restore-dump.adoc:152]: I wonder if the path shouldn't point to the artifact as it does in the `neo4j-admin database restore` command.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1594.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-backup)
- [Restore a database dump](https://neo4j-docs-operations-1594.surge.sh/operations-manual/5/backup-restore/restore-dump)

File: modules/ROOT/pages/backup-restore/restore-dump.adoc (hunk 13 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/backup-restore/restore-dump.adoc:152]: tbh, I think it's redundant in this case

**Pull request merged**